setup.py

```diff
@@ -38,7 +33,7 @@ def get_version():
     author="Chad Smith",
     author_email="grassfedcode@gmail.com",
     description="Install and Run Python Applications in Isolated Environments",
-    long_description=README,
+    long_description=Path(CURDIR, "README.me").read_text(),
```

Comment:
suggest
```
(CURDIR / README.md).read_text(encoding="utf-8", )
```

Reply:
Personally I prefer `CURDIR.joinpath("README.md")` in this kind of situations (but ultimately they are all implemented the same anyway, so it’s just a style issue).

Reply:
`git commit --amend --no-edit` `!=` `git commit setup.py --amend --no-edit` :man_facepalming: 
@uranusjr that is what I have on my workstation. #git-fail